src/browser_action/vitals.js

```diff
@@ -199,6 +199,12 @@
  * Fetches Web Vitals metrics via WebVitals.js
  */
   function fetchWebPerfMetrics() {
+    // web-vitals.js doesn't have a way to remove previous listeners, so we'll save whether
+    // we've already installed the listeners before installing them again.
+    // See https://github.com/GoogleChrome/web-vitals/issues/55.
+    if (self._hasInstalledPerfMetrics) return;
+    self._hasInstalledPerfMetrics = true;
```

Comment:
Great catch! 🐟 ⚓️

I've experimented with this fix and CPU utilization appears significantly lower than with the current release. Appreciate you digging in and discovering this leak, @patrickhulce! Will share more testing shortly.